app/swagger/swagger/requests/profile.rb

```diff
@@ -1415,6 +1415,113 @@ class Profile
           end
         end
       end
+
+      swagger_path '/v0/profile/contacts' do
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::ForbiddenError
+
+          key :description, 'Create a Next-of-Kin or Emergency Contact'
+          key :tags, [:profile]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Contact attributes'
+            key :required, true
+
+            schema do
+              key :$ref, :Contacts
+            end
+          end
+
+          response 201 do
+            key :description, 'Contact created'
+          end
+
+          response 422 do
+            key :description, 'Unprocessable entity'
+            schema do
+              key :$ref, :Errors
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/profile/contacts/{id}' do
```

Comment:
That's a good question. The source system doesn't reference contacts by id, it references them via user credentials and contact_type. Rails, by default, wants to reference records by id (e.g. DELETE /v0/profile/contacts/{id}). Should we repurpose id in this case to refer to the index of VAProfile::Models::AssociatedPerson::PERSONAL_HEALTH_CARE_CONTACT_TYPES, since these will be unique to the user? If so, should we change the parameter name as well? (e.g. - contact_type_index)

Reply:
I think that using the contact type makes sense, as there can be only one _active_ contact per type. We may need to be on the lookout for if/when VA Profile might return _inactive/expired_ contacts, though hopefully that won't happen.

Could we do `/v0/profile/contacts/{contact_type_slug}`, so a request could be `/v0/profile/contacts/emergency-contact`, or do rails conventions frown upon such things?

Reply:
A slug would be perfectly acceptable